settings/filesystem.settings.php

```diff
@@ -5,7 +5,13 @@
  * Contains filesystem settings.
  */
 
+// Local file paths.
 $settings['file_public_path'] = "sites/$site_dir/files";
+$settings['file_private_path'] = "$repo_root/files-private/$site-dir";
```

Comment:
Just going to ask the question... should line 23 below get the same treatment? If so, how do we make this change and keep it non-disruptive to anyone with a site in prod?
Even this change... it says "local file paths", but for anyone not on Acquia hosting, this is changing their file path.